.github/workflows/test-push.yml

```diff
@@ -127,14 +134,16 @@ jobs:
           cd istio-1.*
           export PATH=$PWD/bin:$PATH
           istioctl install --set profile=demo -y
-      - run:  python tests/run_all_tests.py --type=k8s_live_general
+      - run: |
+          export PYTHONPATH=.
+          python tests/run_all_tests.py --type=k8s_live_general
   rest-api:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
       - uses: ./.github/actions/setup-nca-env
       - run: |
-          python network-config-analyzer/nca.py --daemon &
+          python -m nca --daemon &
```

Comment:
Right. The `-m` flag works a bit differently.
* Without `-m`, when you provide a `.py` file to the python executable, `sys,path` will contain all paths in PYTHONPATH and in addition the directory of the provided `.py` file. In our case, since `run_all_tests.py` doesn't reside in the root directory, we must add the root directory to PYTHONPATH or otherwise `run_all_tests.py` will not be able to locate its imports.
* When using the [`-m` flag](https://docs.python.org/3/using/cmdline.html#cmdoption-m), you provide a module/package name, rather than a `.py` file. This will cause `sys.path[0]` to be the empty string, [which directs Python to search modules in the current directory first](https://docs.python.org/3/library/sys.html#sys.path). And that's why it works without the export.